app/server/lib/DocApi.ts

```diff
@@ -1734,11 +1733,24 @@ export class DocWorkerApi {
     return docAuth.docId!;
   }
 
-  private _getDownloadOptions(req: Request, name: string): DownloadOptions {
+  private async _getDownloadFilename(req: Request, tableId?: string): Promise<string> {
+    const addContext = process.env.GRIST_INCLUDE_CONTEXT_TO_DOWNLOAD_FILENAMES || false;
```

Comment:
You'd probably better use `isAffirmative()` here: if `GRIST_INCLUDE_CONTEXT_TO_DOWNLOAD_FILENAMES` is set to `"false"`, `addContext` would be truthy

Reply:
weel seen